decidim-admin/app/views/decidim/admin/components/_form.html.erb

```diff
@@ -86,3 +86,4 @@
 </div>
 
 <%= javascript_include_tag "decidim/admin/form" %>
+<%= javascript_include_tag "decidim/proposals/admin/component_settings" %>
```

Comment:
@lahdeero I think this needs to be added to the `precompile_assets` section, as in other PRs!

Reply:
Sorry! I moved require to form